doc/hooks.conf

```diff
@@ -0,0 +1,75 @@
+/*
+    Example config:
+    Sharding json from one queue into two
```

Comment:
I assume this can do more than two target databases?

Reply:
This scheme can scale to (2^32) databases. We're more likely to run into problems with max pthread or libconfig before we reach that point.